extensions/adobe/experience/customerJourneyManagement/ajo-entity.schema.json

```diff
@@ -86,6 +86,16 @@
               "title": "Channel's variant ID",
               "type": "string",
               "description": "Is not frozen and might remain same between different versions of message publication."
+            },
+            "https://ns.adobe.com/experience/customerJourneyManagement/ajo/channel/_id": {
```

Comment:
This should be a reference as used here

https://github.com/adobe/xdm/blob/6a0ad620e9fcbb8927dc36be6b03dfe59112d8ed/extensions/adobe/experience/customerJourneyManagement/messageprofile.schema.json#L26-L32

Reply:
As discussed. Our channel info is slightly different from `xdm/channels/channel`. So we are only keeping a soft reference to `channel._id` instead of whole `channel` reference.